QUEST.tsv

```diff
@@ -300,203 +300,203 @@ QUEST_20150317_000299	No. Try again.
-xQUEST_20150317_000486	You must be out of your mind going around the gorge without antidotes. {nl}Here, take it.
+xQUEST_20150317_000480	Antidote? {nl}Of course I have some.
+xQUEST_20150317_000481	I used up what I had… {nl}But the people across may have it.
+xQUEST_20150317_000482	You came here without antidotes? {nl}Here, take this.
+xQUEST_20150317_000483	I ran out. {nl}Maybe others have some.
+xQUEST_20150317_000484	I have antidotes. {nl}I'll wish for Dezek's well being.
+xQUEST_20150317_000485	Dezek's lucky. {nl}Usually, they're just eaten.
+xQUEST_20150317_000486	You must be out of your mind going around the gorge without antidotes. {nl}Here, take this.
 QUEST_20150317_000487	I used up all I had a few days ago. {nl}Maybe some other people have it.
-QUEST_20150317_000488	It's a good day for Scorpios to show up. {nl}Damp and dark.
-QUEST_20150317_000489	There are many scorpions here. {nl}I get chills on my neck when I hear a crack under my feet.
-xQUEST_20150317_000490	Be careful of the rainy dusk like now. {nl}Scorpios like to show up at times like these.
-QUEST_20150317_000491	Scorpios are venomous nowadays. {nl}You must be careful.
-QUEST_20150317_000492	Rain again. {nl}The Scorpios might start moving around.
+QUEST_20150317_000488	It's a good day for Scorpio to show up. {nl}Damp and dark.
+xQUEST_20150317_000489	There are many scorpions here. {nl}I get chills on my neck whenever I something cracks underfoot.
+xQUEST_20150317_000490	Be careful of the rainy dusks like now. {nl}Scorpio tend to show up around these times.
+xQUEST_20150317_000491	Scorpio are venomous nowadays. {nl}You have to be careful.
+xQUEST_20150317_000492	Rain again. {nl}The Scorpio must be moving around now.
 QUEST_20150317_000493	Carved writings
-xQUEST_20150317_000494	I have prepared five riddles so that our enemy can't get their hands on it. {nl}But you can't pass it  either if you are not prepared.
-QUEST_20150317_000495	Find it. The road will open when it is proven that you are the true Revelator.
-QUEST_20150317_000496	What makes you walk towards the light in a place full of darkness?
-xQUEST_20150317_000497	The harder you try to find this, the stronger you become, but if you don't find it, you will become weak. {Nl}What is this?
+xQUEST_20150317_000494	I have prepared five dilemmas so that our enemy can't get their hands on it. {nl}But neither of you can pass it if you are not prepared.
+xQUEST_20150317_000495	Find it. The road will open when it has been proven that you are the true Revelator.
+xQUEST_20150317_000496	What makes you walk towards the light in a place full of darkness?
+xQUEST_20150317_000497	The harder you try to find it, the stronger you become, but if you don't find it, you become weak. {nl} What is it?
 QUEST_20150317_000498	Aide de camp Selina
-xQUEST_20150317_000499	Our days to stay self-sufficient is growing. {nl}We even joke about plowing a farm ourselves,  {nl}but it's not becoming a joke anymore.
```

Comment:
I think the old version of the line is more accurate. It's more about the guys staying around longer than they had originally thought.